### PR TITLE
St 75 savings

### PR DESCRIPTION
**Pull request by Froom2**

### CODE REVIEW

File: app/uk/gov/hmrc/tai/viewModels/estimatedIncomeTax/DetailedIncomeTaxEstimateViewModel.scala [hunk 1]
```diff
@@ -227,4 +227,39 @@ with Dividends{
     }
   }
 
+
+  def savingsDescription1(savingsBands: Seq[TaxBand])(implicit messages: Messages): String = {
+
+    val isStartingRate = savingsBands.exists(_.bandType == "SR")
```
Comment (RaminEsfandiari) [inline app/uk/gov/hmrc/tai/viewModels/estimatedIncomeTax/DetailedIncomeTaxEstimateViewModel.scala]: Can "SR" be converted to a constant?

File: app/uk/gov/hmrc/tai/viewModels/estimatedIncomeTax/DetailedIncomeTaxEstimateViewModel.scala [hunk 2]
```diff
@@ -227,4 +227,39 @@ with Dividends{
...
+
+    val startingRateAllowance = savingsBands.find(_.bandType == "SR").flatMap(_.upperBand).getOrElse(0)
+    val personalSavingsAllowanceIncome: BigDecimal = savingsBands.find(_.bandType == "PSR").map(_.income).getOrElse(0)
```
Comment (RaminEsfandiari) [inline app/uk/gov/hmrc/tai/viewModels/estimatedIncomeTax/DetailedIncomeTaxEstimateViewModel.scala]: How about "PSR"?

File: app/views/estimatedIncomeTax/detailedIncomeTaxEstimate.scala.html [hunk 3]
```diff
@@ -94,10 +94,18 @@ <h2 id="tax-on-dividend-income-text" class="heading-section">@Messages("tai.esti
                 </section>
 
                 <section class="section soft--top">
-                    <h3 id="tax-on-savings-text" class="heading-medium heading-section">@Messages("tai.estimatedIncome.detailedEstimate.savingsInterest.subHeading")</h3>
-                    <p id="tax-on-savings-desc">
-                        @Messages("tai.estimatedIncome.savings.desc")
+                    <h2 id="tax-on-savings-text" class="heading-medium heading-section">@Messages("tai.estimatedIncome.detailedEstimate.savingsInterest.subHeading")</h2>
+                    <p id="tax-on-savings-desc1">
+                        @DetailedIncomeTaxEstimateViewModel.savingsDescription1(viewModel.savings)
```
Comment (RaminEsfandiari) [inline app/views/estimatedIncomeTax/detailedIncomeTaxEstimate.scala.html]: May I ask why we are using the ViewModel in this way?

File: app/uk/gov/hmrc/tai/viewModels/estimatedIncomeTax/DetailedIncomeTaxEstimateViewModel.scala (hunk 1 above)
Comment (Froom2) [reply app/uk/gov/hmrc/tai/viewModels/estimatedIncomeTax/DetailedIncomeTaxEstimateViewModel.scala]: Yeah there are constants for this!

**Comment by RaminEsfandiari** [comment]
Can you also fix the title of the PR please?

**Pull request merged**